docs/release_process.md

```diff
@@ -12,52 +11,120 @@ Everyone is welcome to test the stabilization branch and propose fixes.
+    - Propose bug fixes by targeting the PRs to the **stabilization** branch. e.g. `stabilization-v0.1.65`
+    - Once the PR is **merged** in the **stabilization** branch, cherry-pick it to the **master** branch.
+    For example:
+    ```
+    git checkout master
+    git pull upstream master
+    git checkout -b backport_from_stabilization
+    git cherry-pick abcd1234
+    ...
+    ```
+    _NOTE: There are different ways to backport a commit from the stabilization to the master branch._
+    _The above approach is just one example._
+
+In the past, to ensure that all fixes pushed to the stabilization branch were also included in the
+master branch, every Friday during the stabilization phase, a PR was created to merge the
+stabilization changes into the master branch. However, this approach was prone to conflicts that
+could be complex to be solved after one week of asynchronous contributions on master branch.
+Therefore, since the `0.1.63` release, a simpler and more reliable process has been adopted where
+the authors themselves ensure that the fixes are merged into each branch at almost the same time.
+This reduces the risk of conflicts and also makes it easier for authors to resolve them as the
+changes are still "fresh".
+
+### Tests
+
+There is a GitHub Action hooked up with **stabilization** branch that will run a set of tests on every push.
+Make sure that all these tests are passing before moving on with the release process.
 
-Tests during Stabilization Phase
------------
+# Release
 
-There is a GitHub Action hooked up with **stabilization** branch that will run a set of tests on every push.\
-Make sure that all these tests are passing before moving on with the release process.
+## Updating Contributors List
 
-Before the Release
-------------------
+- Update the contributors list by accessing the root folder of Content repository, in the
+**stabilization** branch, and executing the following command:
+    ```
+    PYTHONPATH=. utils/generate_contributors.py`
+    ```
+    - De-duplicate names if necessary.
+    - Make a commit send the a PR.
+    - Reference: https://github.com/ComplianceAsCode/content/pull/9601
 
--   Make sure the version in `CMakeLists.txt` is correct, i.e.: the
-    version corresponds to an unreleased version number.
+## Checklist
 
--   Run `PYTHONPATH=. utils/generate_contributors.py` to update the
-    contributors list. De-duplicate names if necessary.\
-    Make a commit and PR it.
-    * Make sure you don’t have any uncommited changes, otherwise they may be lost during the release.
-    * Make sure you have the `master` and `stabilization` branch checked out and up to date.
+- Make sure the version in `CMakeLists.txt` is correct.
+    - The version should correspond to an unreleased version number.
+- Make sure you don’t have any uncommitted changes, otherwise they may be lost during the release.
```

Comment:
```
- Make sure you don’t have any uncommitted changes, otherwise they may be lost during the release.
- Make sure you have the **stabilization** branch checked out and up to date.
```
These items were part of the process to update the contributors list, it doesn't make sense in the `Checklist`.

Reply:
Ok, so it can also be simplified if the contributors list is updated before the stabilization branch.